.github/workflows/sim.yaml

```diff
@@ -44,6 +44,8 @@ jobs:
         - "sig-rsa validate-primary-slot ram-load multiimage"
         - "sig-rsa validate-primary-slot direct-xip multiimage"
         - "sig-ecdsa hw-rollback-protection multiimage"
+        - "sig-ecdsa-psa psa-crypto-api"
```

Comment:
no need to specify `psa` twice
how about:

```
sig-ecdsa psa-crypto-api sig-p256
sig-ecdsa psa-crypto-api sig-p384
```

or

```
sig-ecdsa-psa sig-p256 
sig-ecdsa-psa sig-p384
```

the latter fits better with the previous scheme and the `psa-crypto-api` feature should then be removed

Reply:
Hi, thanks for the suggestions. I've implemented the second approach as it fits the current scheme more, but I opted not to add an algorithm specifier for p256 as that's the default.